code/_compile_options.dm

```diff
@@ -1,5 +1,6 @@
 //#define TESTING //By using the testing("message") proc you can create debug-feedback for people with this
 								//uncommented, but not visible in the release version)
+#define LOWMEMORYMODE
```

Comment:
remember to undefine this when we need to merge this

Reply:
yeah, I'll leave this conversation open for it too. I think it causes integration tests to fail as well anyways, though.